ramls/holdings-sync.raml

```diff
@@ -0,0 +1,40 @@
+#%RAML 1.0
+title: Inventory Storage Holdings Batch Sync API
+version: v1.0
```

Comment:
This version should match the one used in the module descriptor. I believe that was recently changed to 0.1. If so, please update the RAML files as well.

Reply:
Done.